build.sbt

```diff
@@ -113,9 +118,14 @@ lazy val core = crossProject(JVMPlatform, JSPlatform)
     },
     mimaBinaryIssueFilters ++= Mima.ignoredABIProblems,
     libraryDependencies += "org.scalameta" %% "testkit" % "3.7.3" % Test
+  ).nativeSettings(
+    nativeLinkStubs := true,
+    scalaVersion := scala211,
+    crossScalaVersions := List(scala211)
   )
 lazy val coreJVM = core.jvm
 lazy val coreJS = core.js
+lazy val coreNative = core.native
```

Comment:
🎉

Reply:
We need to wait for 0.3.9 so I'm thinking it would be better to just get that released and then work upstream. I have a PR for `sconfig` ready. https://github.com/ekrich/sconfig/pull/19